java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -613,4 +614,79 @@ CompletableFuture<Map<String, Double>> zrangeWithScores(
      * }</pre>
      */
     CompletableFuture<Long> zremrangebylex(String key, LexRange minLex, LexRange maxLex);
+
+    /**
+     * Returns the union of members from sorted sets specified by the given <code>keys</code>.
+     *
+     * @see <a href="https://redis.io/commands/zunion/">redis.io</a> for more details.
+     * @param keys The keys of sorted sets.
+     * @param options Weight and Aggregate options.
+     * @return The resulting sorted set from the union.
+     * @example
+     *     <pre>{@code
+     * WeightAggregateOptions options =
+     *     WeightAggregateOptions.builder()
+     *             .aggregate(Aggregate.MAX)
+     *             .weights(List.of(1.0, 2.0))
+     *             .build();
+     * String[] payload = client.zunionstore("newSortedSet", new String[] {"mySortedSet1", "mySortedSet2"}, options).get()
+     * assert payload.equals(new String[] {"elem1", "elem2", "elem3"});
+     * }</pre>
+     */
+    CompletableFuture<String[]> zunion(String[] keys, WeightAggregateOptions options);
+
+    /**
+     * Returns the union of members from sorted sets specified by the given <code>keys</code>.<br>
+     * To perform a zunion operation while specifying custom weights and aggregation settings, use
+     * {@link #zunion(String[], WeightAggregateOptions)}
```

Comment:
You have 4 functions, I think you need 3 links. At least 2 - to function with aggregation and to function with scores